### PR TITLE
Refactoring scripts

### PR DESCRIPTION
**Pull request by deveusss**

### CODE REVIEW

**Review by deveusss — COMMENTED** [review]
Added more scripts

File: contracts/cw-croncat/scripts/testnet_create_recurring_task.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+. ./testnet_init_vars.sh
```
Comment (Buckram123) [inline contracts/cw-croncat/scripts/testnet_create_recurring_task.sh:1]:
```suggestion
#!/bin/bash
. ./testnet_init_vars.sh
```

File: contracts/cw-croncat/scripts/testnet_create_staking_task.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+. ./testnet_init_vars.sh
```
Comment (Buckram123) [inline contracts/cw-croncat/scripts/testnet_create_staking_task.sh:1]:
```suggestion
#!/bin/bash
. ./testnet_init_vars.sh
```

File: contracts/cw-croncat/scripts/testnet_deploy.sh [hunk 3]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-sh build.sh
+sh ../build.sh
```
Comment (Buckram123) [inline contracts/cw-croncat/scripts/testnet_deploy.sh:4]:
```suggestion
```
No reason to build it right before building with docker (the only difference is clippy) and extra time

File: contracts/cw-croncat/scripts/testnet_get_agent_ids.sh [hunk 4]
```diff
@@ -0,0 +1,5 @@
+. ./testnet_init_vars.sh
```
Comment (Buckram123) [inline contracts/cw-croncat/scripts/testnet_get_agent_ids.sh:1]:
```suggestion
#!/bin/bash
. ./testnet_init_vars.sh
```

File: contracts/cw-croncat/scripts/testnet_get_agent_tasks.sh [hunk 5]
```diff
@@ -0,0 +1,6 @@
+. ./testnet_init_vars.sh
```
Comment (Buckram123) [inline contracts/cw-croncat/scripts/testnet_get_agent_tasks.sh:1]:
```suggestion
#!/bin/bash
. ./testnet_init_vars.sh
```

File: contracts/cw-croncat/scripts/testnet_get_contract_info.sh [hunk 6]
```diff
@@ -0,0 +1,3 @@
+. ./testnet_init_vars.sh
```
Comment (Buckram123) [inline contracts/cw-croncat/scripts/testnet_get_contract_info.sh:1]:
```suggestion
#!/bin/bash
. ./testnet_init_vars.sh
```

File: contracts/cw-croncat/scripts/testnet_get_tasks.sh [hunk 7]
```diff
@@ -0,0 +1,5 @@
+. ./testnet_init_vars.sh
```
Comment (Buckram123) [inline contracts/cw-croncat/scripts/testnet_get_tasks.sh:1]:
```suggestion
#!/bin/bash
. ./testnet_init_vars.sh
```

File: contracts/cw-croncat/scripts/testnet_init_vars.sh [hunk 8]
```diff
@@ -0,0 +1,2 @@
+NODE="--node https://rpc.uni.juno.deuslabs.fi:443"
```
Comment (Buckram123) [inline contracts/cw-croncat/scripts/testnet_init_vars.sh:1]:
```suggestion
#!/bin/bash
NODE="--node https://rpc.uni.juno.deuslabs.fi:443"
```

File: contracts/cw-croncat/scripts/testnet_query_agents.sh [hunk 9]
```diff
@@ -0,0 +1,5 @@
+. ./testnet_init_vars.sh
```
Comment (Buckram123) [inline contracts/cw-croncat/scripts/testnet_query_agents.sh:1]:
```suggestion
#!/bin/bash
. ./testnet_init_vars.sh
```

File: contracts/cw-croncat/scripts/testnet_reg_and_proxy_call.sh [hunk 10]
```diff
@@ -0,0 +1,17 @@
+. ./testnet_init_vars.sh
```
Comment (Buckram123) [inline contracts/cw-croncat/scripts/testnet_reg_and_proxy_call.sh:1]:
```suggestion
#!/bin/bash
. ./testnet_init_vars.sh
```

File: contracts/cw-croncat/scripts/testnet_unregister_agent.sh [hunk 11]
```diff
@@ -0,0 +1,5 @@
+. ./testnet_init_vars.sh
```
Comment (Buckram123) [inline contracts/cw-croncat/scripts/testnet_unregister_agent.sh:1]:
```suggestion
#!/bin/bash
. ./testnet_init_vars.sh
```

**Review by Buckram123 — CHANGES_REQUESTED** [review]

**Review by Buckram123 — APPROVED** [review]
Nice

**Review by Buckram123 — APPROVED** [review]

**Review by Buckram123 — APPROVED** [review]

**Review by NinoLipartiia — APPROVED** [review]

**Review by Buckram123 — APPROVED** [review]

File: contracts/cw-croncat/scripts/testnet_create_staking_task.sh [hunk 12]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+. ./testnet_init_vars.sh
+USER=juno1pd43m659naajmn2chkt6tna0uud2ywyp5dm4h3
+CONTRACT=juno12z4hh9r3j9aurjn6ppkgyjrkuu4ugrdectsh792w8feyj56dhlssvntdls
```
Comment (mikedotexe) [inline contracts/cw-croncat/scripts/testnet_create_staking_task.sh:4]: Eventually, would be great to have these be given as parameters, like here:
https://www.redhat.com/sysadmin/arguments-options-bash-scripts
See the section on "Positional parameters"

**Review by mikedotexe — APPROVED** [review]

File: contracts/cw-croncat/scripts/testnet_create_staking_task.sh (hunk 12 above)
Comment (deveusss) [reply contracts/cw-croncat/scripts/testnet_create_staking_task.sh:4]: good point)

**Review by mikedotexe — APPROVED** [review]

**Pull request merged**